bin.src/ci_imsim_run.py

```diff
@@ -91,7 +91,6 @@ def run(self, currentState: BuildState):
             "-p", "$DRP_PIPE_DIR/pipelines/LSSTCam-imSim/DRP-ci_imsim.yaml",
             "--skip-existing",
             "--save-qgraph", os.path.join(self.runner.RunDir, QGRAPH_FILE),
-            "--config", f"calibrate:deblend.useCiLimits={not self.arguments.no_limit_deblend}",
```

Comment:
Is this not implemented in the new task? I vaguely remember it didn't work right anyway and only the multiband one was really necessary (to limit Scarlet runtime, which might also be moot now).